sr_example/scripts/sr_example/sr_mix_examples_unsafe.py

```diff
@@ -10,6 +10,14 @@
 hand_commander = SrHandCommander()
 arm_commander = SrArmCommander()
 
+rospy.loginfo("Set arm teach mode ON")
+arm_commander.set_teach_mode(True)
+# sleep for some time during which the arm can be moved around by pushing it
+# but be careful to get away before the time runs out. You are warned
+rospy.sleep(20.0)
```

Comment:
Mmm  not sure about this. As a general behaviour it would look a bit weird. They might want to switch to teach mode and stay there,  or switch to teach mode and then retrieve and store positions. Or as you suggested, use tactile data to activate deactivate tech mode. For these you need the function to be non blocking.